src/explore-education-statistics-common/src/services/tableBuilderService.ts

```diff
@@ -275,21 +275,26 @@ const tableBuilderService = {
   async getDataBlockTableData(
     releaseId: string,
     dataBlockParentId: string,
-    boundaryLevelId?: number | undefined,
   ): Promise<TableDataResponse> {
-    return boundaryLevelId
-      ? dataApi.get(
-          `/tablebuilder/release/${releaseId}/data-block/${dataBlockParentId}?boundaryLevelId=${boundaryLevelId}`,
-        )
-      : dataApi.get(
-          `/tablebuilder/release/${releaseId}/data-block/${dataBlockParentId}`,
-        );
+    return dataApi.get(
+      `/tablebuilder/release/${releaseId}/data-block/${dataBlockParentId}`,
+    );
   },
   getFastTrackTableAndReleaseMeta(
     dataBlockParentId: string,
   ): Promise<FastTrackTableAndReleaseMeta> {
     return dataApi.get(`/tablebuilder/fast-track/${dataBlockParentId}`);
   },
+  getGeoJson(
+    releaseId: string,
+    dataBlockParentId: string,
+    boundaryLevelId: number,
+  ): Promise<Dictionary<LocationOption[]>> {
```

Comment:
Could do a bit better with this type. `LocationOption` has an optional `geoJson` property, but as we're requesting from this endpoint, we know that the `geoJson` is populated.

We could do something like:

```ts
// tableBuilderService.ts

export interface LocationGeoJsonOption extends LocationOption {
  geoJson: GeoJsonFeature;
}


const tableBuilderService = {
  getLocationGeoJson(...): Promise<Dictionary<LocationGeoJsonOption[]>> {
    // ...
  }
}
```